### ISSUE TITLE
DAY NOT FİLL

### ISSUE THREAD

**Issue by akbaba3141** [omkarcloud/outlook-account-generator#6]
does not fill in the 'day' on your birthday

https://imgur.com/a/z5uf42h

**Comment by Chetan11-dev** [comment]
Does it happen some times or every time?

**Comment by pacokeks** [comment]
Happens everytime for me. Setup is german for me tho due to my location. Everything else works just fine till then

**Comment by gameuser1982** [comment]
I am having the same exact problem

**Comment by gameuser1982** [comment]
Yeah it seems if your country is not set as United Stated (and it is set based on ip) then this bot hangs on day. I confirmed it by testing with a US VPN but the US VPN ip is known and it's asking for phone verification
